### ISSUE TITLE
Read the first map packet to confirm that login is complete instead of sleeping for 100ms

### ISSUE THREAD

**Issue by divinity76** [Znote/POTCP#11]
sometimes you want the login to be really fast if logging in with multiple characters at the same time, hence the

```php
            // if we don't sleep a little after logging in, nothing will work, talking, walking, etc won't respond for the first
            // few milliseconds or so. (???)
            usleep(100 * 1000);
```

part in Tibia_client_internal::login() should probably be optional (but on by default, because in MOST situations, it's a good thing) for example, that usleep makes the class unsuitable for making [magebombs](https://youtu.be/qB07bAPowCo?t=9)

- that said, an even better solution than the usleep would be to read the first map packet to confirm that login is complete. then the login speed wouldn't be "100 milliseconds", it would be the speed of the server and your bandwidth (as it probably *should* be)

**Comment by Znote** [comment]
Yeah, for now I think #6 should be fixed first. Once we get map packet parsing going we can wait for first map packet instead of sleeping.
